### DISCUSSION TITLE
How to create a nicely-looking "default" tag?

### DISCUSSION THREAD

**Discussion by InSyncWithFoo** [squidfunk/mkdocs-material#7036] [Q&A] [answered]
And by that I mean something like this:

![](https://github.com/squidfunk/mkdocs-material/assets/122007197/734bfbce-eead-4153-a21d-39ee3a66b394)

A quick search leads to [this file](https://github.com/squidfunk/mkdocs-material/blob/cd086110bfd65eb6dfeed8c3a01f500e9cdb7468/src/overrides/hooks/shortcodes.py#L36), which doesn't look like something I can reuse for my project. Is there an equivalent?

**Comment by alexvoss** [comment, marked as answer]
This has been asked a few times before, for example here: https://github.com/squidfunk/mkdocs-material/discussions/6724

These shortcodes are not part of Material for MkDocs but are a customization used for the documentation. You can add a [hook](https://www.mkdocs.org/user-guide/configuration/#hooks) like the one you referenced to your own project.

**Comment by Aetherinox** [comment]
To add on to the other answer, yes, those little tags aren't part of mkdocs, but just to [show you an example](https://aetherinox.github.io/obsidian-gistr/settings/github/#codeblock-background-color); they can be integrated if you know a little python.

I simply took the shortcodes.py file from https://github.com/squidfunk/mkdocs-material/blob/master/src/overrides/hooks/shortcodes.py and then modified it to fit the tags and badges I needed. And even added an "example color" block to show the default colors like in the example link above.

**Reply by InSyncWithFoo** [reply]
Thanks; I'll refer to it when I revisit this. By the way, the color contrast is quite bad in light mode.

**Reply by Aetherinox** [reply]
Yeah light mode needs to be modified. Dark mode is what I use 99.9% of the time. I think light mode in general just blinds me, so I keep it on dark lol
